diff --git a/SplashView.tsx b/SplashView.tsx
index ca6e1dfa91ac0f9bf377f7fabff46ef70d414ab7..b7f8d7dd5099648fd08068134c7bf0b52e41e63c 100644
--- a/SplashView.tsx
+++ b/SplashView.tsx
@@ -235,7 +235,6 @@ const SplashView = ({
         fit={fit}
         alignment={alignment}
         autoplay
-        loopMode={loopMode}
         onError={(err) => {
           console.error(`${err.type}: ${err.message}`);
         }}
